.github/workflows/create-draft-release.yml

```diff
@@ -0,0 +1,65 @@
+name: Create Draft Release
+
+on:
+  push:
+    branches:
+    - main
+
+jobs:
+  smoke:
+    name: Smoke Test
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Install Pack
+      run : |
+        (curl -sSL "https://github.com/buildpacks/pack/releases/download/v0.13.1/pack-v0.13.1-linux.tgz" | sudo tar -C /usr/local/bin/ --no-same-owner -xzv pack)
```

Comment:
Is there a reason we're pinning this specific version of pack? I wonder if there's a way to grab the latest stable version in a less brittle way (e.g. version 14 is now out, do we want to use that one?)

Reply:
Pretty sure this is just what we do in all of our workflows. I had the same thought and have an action item to create an action that'll grab latest eventually

Reply:
It probably makes sense to bump this pinned version to v14 as you suggested though at the very least for now

Reply:
If you'd like the latest pack, we should be able to leverage the `buildpacksio/pack` image published by the pack maintainers. Something like the following to run pack commands:
```
    - name: Create-builder
      uses: docker://buildpacksio/pack
      with:
        args: create-builder testbuilder --config builder-repo/builder.toml
```

Reply:
I think that's a solid idea @arjun024.